Lib/fontmake/font_project.py

```diff
@@ -175,19 +182,20 @@ def save_otfs(
     def subset_otf_from_ufo(self, otf_path, ufo):
         """Subset a font using export flags set by glyphsLib."""
 
-        font_lib_prefix = 'com.schriftgestaltung.'
+        font_lib_prefix = GLYPHS_PREFIX
```

Comment:
Maybe. Though this way the prefixes for the font lib and glyph lib are more explicitly distinguished. We could do without the `glyph_lib_prefix` variable also?